MIGRATION.md

```diff
@@ -70,37 +75,93 @@
 
 ## How to migrate your packages to v4
 
-Install a package that contains all of the components from Forma 36. Tree-shaking will take care of your build, so it will include only components that you use.
-We separate only icons from the main package. If you need to use our icons, install them separately, by running the following commands:
-:
+### Step 1 - install new packages
+
+Install a package that contains all of the components from Forma 36 version 4. Tree-shaking will take care of your build, so it will include only components that you use.
 
 For NPM
 
-```sh
-npm install @contentul/f36-components@beta
-npm install @contentful/f36-icons@beta
+```bash
+npm install @contentul/f36-components
+npm install @contentful/f36-tokens
 ```
 
 For YARN
 
-```sh
-yarn add @contentul/f36-components@beta
-yarn add @contentful/f36-icons@beta
+```bash
+yarn add @contentul/f36-components
+yarn add @contentful/f36-tokens
+```
+
+#### Install icons package separately
+
+Starting from version 4, icons are not included in the main components package. If you need to use our icons or you use them already in version 3, install them separately, by running the following commands:
+
+```bash
+npm install @contentul/f36-icons
 ```
 
-You can install separate packages and use chosen components separately.
-For example, if you need only the `Button` component in your project, you can add this package by running the following command:
+For YARN
+
+```bash
+yarn add @contentul/f36-icons
+```
+
+#### Install packages separately
+
+If for some reason, your project requires installing packages separately it is possible in version 4.
+For example, if you need only the `Button` component in your project, you can add only this package by running the following command:
+
+For NPM
+
+```bash
+npm install @contentul/f36-button
+```
+
+For YARN
+
+```bash
+yarn add @contentul/f36-button
+```
+
+Note: We don't recommend using it this way, it is much easier to [install full package of components](#Step-1-install-new-packages). Tree-shaking will take care of your build
```

Comment:
```suggestion
<Note> We don't recommend using it this way, it is much easier to install full package of components. Tree-shaking will take care of your build </Note>
```
You can use React components in mdx (It's how the Props table is rendered) so what do you think about using Note here? https://v4-forma-36.netlify.app/components/note/
You would have to use TexLink too for that link

Reply:
Another option would be to use Markdown quote like:

> This is a quote

but I think our MDXRenderer is not prepare to render one of those so we would need to work a bit on it
(Maybe that's a job for another PR 🤷 )

Reply:
uuu! I like the idea about the Note component!